src/index.html

```diff
@@ -11,7 +11,146 @@
+            class = "inpMb"
+          >
+        </label>
+        <label class="formField">How old are You?
+          <input
+            type = "number"
+            name = "age"
+            value = "12"
+            min = "1"
+            max = "100"
+            class = "inpMb"
+            required
+          >
+        </label>
+        <label class="formField">Full date of birth:
+          <input
+            type = "date"
+            name = "date"
+            class = "inpMb"
+            required
+          >
+        </label>
+        <label class="formField">I accept the term of the agreement
+          <input
+            type = "checkbox"
+            name = "checkTerm"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class = "group">
+        <legend>Registration</legend>
+        <label class="formField">E-mail:
+          <input
+            type = "email"
+            name = "email"
+            required
+            class = "inpMb"
+            placeholder = "email@example.com"
+          >
+        </label>
+        <label class="formField">Password:
+          <input
+            type = "password"
+            name = "password"
+            minlength="5"
+            maxlength="20"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class = "group">
+        <legend>An interesting fact about you!</legend>
+        <div class="formField inpMb">Do you love cats?
+          <label>
+            <input
+              type = "radio"
+              name = "radLoveCats"
+            > Yes
+          </label>
+          <label>
+            <input 
+              type = "radio"
+              name = "radLoveCats"
+            > No
+          </label>
+        </div>
+        <label class = "formField">What is your favorite color?
+          <input
+            type = "color"
+            name = "color"
+            class = "inpMb"
+          >
+        </label>
+        <label class = "formField">What time do you go to bed?
+          <input 
+            type = "time"
+            name = "time"
+            class = "inpMb"
+          >
+        </label>
+        <label class = "formField inpMb">What are you favorite brand of cars?
+          <select
+            name = "selectBradCar"
+            required
+            multiple
+          >
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+          </select>
+        </label>
+        <label class = "formField">How do you rate our work?
+          <input 
+            type = "range"
+            name = "rangeRate"
+            value = "0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class = "group">
+        <legend>Additional info:</legend>
+        <label class = "formField inpMb">Comments:
+          <textarea 
+            name = "commentTextarea"
+            required
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name = "commentTextarea"
            required
          ></textarea>
```

It should fix extra spaces in textarea field

Reply:
<textarea 
      name = "commentTextarea"
      required
></textarea> 

That doesn't work.

error:  Tag’s closing bracket should be on the same level    (closing-and-opening-on-seme-level)
                as opening one
--------------------------------------------------------------------------------------------------------

And in the previous format works: 

<textarea
      name="commentTextarea"
      required
>
</textarea>

Reply:
With textarea your hint doesn't work. Gives an error message:  
Tag’s closing bracket should be on the same level    (closing-and-opening-on-seme-level)
                as opening one
Before the fix it worked.

Reply:
https://github.com/mate-academy/layout_html-form/pull/329/files 
I redid the task.